README.md

```diff
@@ -21,3 +21,9 @@ Here is a list of known category options:
 | `x-tao-option-nextSection` | Enable the next section button |
 | `x-tao-option-nextSectionWarning` | Enable the next section button, display a confirm message |
 | `x-tao-proctored-auto-pause` | Enable autopause before entering the next section |
+
+
+REST API
```

Comment:
Thanks very cool to have on the Read me

Reply:
Kudos to @SergiiTao, who asked around about README structure.